packages/preferences/src/store/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { createReduxStore, registerStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import reducer from './reducer';
+import * as actions from './actions';
+import * as selectors from './selectors';
+import { STORE_NAME } from './constants';
+
+/**
+ * Store definition for the interface namespace.
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/HEAD/packages/data/README.md#createReduxStore
+ *
+ * @type {Object}
+ */
+export const store = createReduxStore( STORE_NAME, {
+	reducer,
+	actions,
+	selectors,
+	persist: [ 'features' ],
+	__experimentalUseThunks: true,
+} );
+
+// Once we build a more generic persistence plugin that works across types of stores
+// we'd be able to replace this with a register call.
```

Comment:
It feels like a good opportunity to basically deprecated the persistence plugin and have its logic handled directly by the current package.

Reply:
(not necessary for the first release of the package)

Reply:
Yep, that would be great. Just checked and it looks like there's one other usage in production - `enableItems` in the interface package.